extensions/security/runtime-spi/src/main/java/io/quarkus/security/spi/runtime/DisabledAuthorizationController.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.security.spi.runtime;
+
+import javax.annotation.Priority;
+import javax.enterprise.inject.Alternative;
+import javax.inject.Singleton;
+import javax.interceptor.Interceptor;
+
+/**
+ * Controller used in dev mode if {@code quarkus.security.authorization.enabled=false}.
+ */
+@Alternative
+@Priority(Interceptor.Priority.LIBRARY_AFTER)
+@Singleton
```

Comment:
Also here I wasn't sure what priority to use and if it would be better to instead use the `@DefaultBean` annotation on `AuthorizationController`.

Reply:
It will be used in devmode only, so this approach looks OK. Lets see what @stuartwdouglas thinks though.

Reply:
I think this is ok.